waffle_hub/hub/adapter/autocare_dlt/configs/model_cfg.py

```diff
@@ -100,6 +100,61 @@ def get_model_config(
             "num_classes": len(categories),
         }
 
+    elif model_type == "TextRecognition":
+        if model_size == "s":
+            backbone = "resnet18"
+        elif model_size == "m":
+            backbone = "resnet34"
+        elif model_size == "l":
+            backbone = "resnet50"
+
+        return {
+            "task": model_type,
+            "model": {
+                "Transformation": {"name": None},
+                "FeatureExtraction": {"name": backbone, "feature_index": 3, "output_size": 256},
+                "SequenceModeling": {"name": "BiLSTM", "input_size": 256, "hidden_size": 256},
+                "Prediction": {"name": "CTC", "input_size": 256},
+                "max_string_length": 50,
+            },
+            "loss": {"str_loss": {"name": "STRCTCLoss", "params": {}}},
+            "optim": {"name": "Adam", "lr": 0.01},
```

Comment:
Why is the `learning_rate` fixed?

Reply:
same question in 147 line.

Reply:
> Why is the `learning_rate` fixed?
> 
> I believed that using a value that I have already tried before would provide more stability in terms of performance. Therefore, I adopted the values that were used with the dyiot dataset.

Reply:
That why we provide the default parameters.

This is hard coding.

Reply:
Oh, I see there was a lr variable. My mistake. I will make the necessary changes.